BankSDK/GiniBankSDKExample/Tests/SettingsViewControllerTests.swift

```diff
@@ -191,7 +191,7 @@ final class SettingsViewControllerTests: XCTestCase {
 		case .pdf_and_images:
 			selectedSegmentIndex = 2
 		}
-		contentData.append(.fileImportType(data: SegmentedOptionModel(selectedIndex: selectedSegmentIndex)))
+        contentData.append(.segmentedOption(data: SegmentedOptionModel(optionType: .fileImport, selectedIndex: selectedSegmentIndex)))
```

Comment:
We should also add `.segmentedOption(data: SegmentedOptionModel(optionType: .entryPoint,...` and because we are adding a second `segmentedOption` in content data then `getEntryPointIndex() ` will not work since the first index indicates segmented control for `fileImport`.